frontend/src/entities/account/account/model/process-multi-transaction.ts

```diff
@@ -0,0 +1,43 @@
+import { Connection, VersionedTransaction } from '@solana/web3.js'
+
+export const processMultiTransaction = async (
+  transactions: string[],
+  wallet: any,
+  connection: Connection,
+  latestBlockhash: { blockhash: string; lastValidBlockHeight: number },
+  operationType = 'Transaction'
+): Promise<{ signatures: string[]; transactions: VersionedTransaction[] }> => {
+  const signatures: string[] = []
+
+  // Deserialize and prepare transactions
+  const deserializedTransactions: VersionedTransaction[] = transactions
+    .map((txData: string) => Buffer.from(txData, 'base64'))
+    .map((txData: Buffer) => VersionedTransaction.deserialize(txData))
+
+  const signedTransactions = await wallet.signAllTransactions!(deserializedTransactions)
+
+  for (const txn of signedTransactions) {
+    console.log('Simulating transaction before sending...')
+    const simulation = await connection.simulateTransaction(txn, {
+      // Default [SimulateTransactionConfig]
```

Comment:
Do we want/need this comment?

Reply:
That's to show that "default" values will be used. We can remove it, but it will look like this:
`connection.simulateTransaction(txn, {})`

Reply:
Oh, I see. Let's use something more intuitive:
```suggestion
      // Empty body: Default [SimulateTransactionConfig]
```

Reply:
Done